js/vote.js

```diff
@@ -48,7 +48,7 @@ function updateCounters() {
 
 function updateAvatar(obj) {
 	// Temporary hack - Check if we have Nextcloud or ownCloud with an anomymous user
-	if (!document.getElementById('nextcloud') && OC.currentUser === '') {
+	if (!document.getElementById('nextcloud') && (OC.currentUser === '' || OC.currentUser === false)) {
```

Comment:
maybe, just using `!OC.currentUser` instead of the `or` ?

Reply:
`!OC.currentUser` does the job in 10.0.7 and 10.0.8